### PR TITLE
Split MavenConversionIntegrationTest

### PR DESCRIPTION
**Pull request by wolfs**
into Kotlin and Groovy DSL to have faster test classes.

### CODE REVIEW

**Comment by wolfs** [comment]
@bot-gradle test QFL

**Comment by wolfs** [comment]
@bot-gradle test and merge

File: subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy [hunk 1]
```diff
@@ -106,14 +107,11 @@ Root project 'webinar-parent'
 +--- Project ':webinar-impl' - Webinar implementation
 \\--- Project ':webinar-war' - Webinar web application
 """
-
-        where:
-        scriptDsl << ScriptDslFixture.SCRIPT_DSLS
     }
 
-    @ToBeFixedForConfigurationCache(because = "Kotlin Gradle Plugin") // Kotlin compilation is used for the pre-compiled script plugin
+    @ToBeFixedForConfigurationCache(because = "Kotlin Gradle Plugin", bottomSpecs = "KotlinDslMavenConversionIntegrationTest") // Kotlin compilation is used for the pre-compiled script plugin
```
Comment (blindpirate) [inline subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy:112]: Why not enable this annotation for `GroovyDslMavenConversionIntegrationTest`? Also, I guess there's no way to make this string a constant, right?

Comment (wolfs) [reply subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy:112]: The test works with configuration caching and Groovy. Only Kotlin fails.

Comment (wolfs) [reply subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/MavenConversionIntegrationTest.groovy]: I extracted the name in a constant.

**Comment by wolfs** [comment]
@bot-gradle test QFL

**Review by blindpirate — APPROVED** [review]
LGTM!

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test QFL

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**